landing-page/types/alces-branding/content/data/bookmarks.yaml

```diff
@@ -0,0 +1,5 @@
+# bookmarks:
+#   # Bookmarks/common directories to include in Flight File Manager
+#   - path: ""
+#     text: ""
+#     fa_icon: ""
```

Comment:
I can't see why we'd want different bookmarks when installing the Flight Direct branding.  Unless you have a suggestion for why we'd want that this file should be removed.